samples/nrf9160/http_update/cert/BaltimoreCyberTrustRoot

```diff
@@ -0,0 +1,81 @@
+/*
```

Comment:
There are two copies certificate files now (the other one in `http_update/application_update/cert`).